pilot/test/integration/infra.go

```diff
@@ -275,7 +275,13 @@ func (infra *infra) deployApps() error {
 	if err := infra.deployApp("c-v2", "c", 80, 8080, 90, 9090, 70, 7070, "v2", true, false); err != nil {
 		return err
 	}
-	return infra.deployApp("d", "d", 80, 8080, 90, 9090, 70, 7070, "per-svc-auth", true, true)
+	if err := infra.deployApp("d", "d", 80, 8080, 90, 9090, 70, 7070, "per-svc-auth", true, true); err != nil {
```

Comment:
Whew, not related to your PR, but we should clean this up with multierror. (Really Pilot needs a small wrapper around multi error that maintains nil-ness of errors; I have such a wrapper in a WIP PR that I'll get out soon and update this too.)

Reply:
:). Ack